pytorch_toolkit/instance_segmentation/configs/instance-segmentation-0091.py

```diff
@@ -0,0 +1,12 @@
+_base_ = '../../../external/mmdetection/configs/dcn/cascade_mask_rcnn_r101_fpn_dconv_c3-c5_1x_coco.py'
+data_root = '../../data/coco/'
+data = dict(
+    train=dict(
+        ann_file=data_root + 'annotations/instances_train2017.json',
+        img_prefix=data_root + 'train2017/'),
+    val=dict(
+        ann_file=data_root + 'annotations/instances_val2017.json',
+        img_prefix=data_root + 'val2017/'),
+    test=dict(
+        ann_file=data_root + 'annotations/instances_val2017.json',
+        img_prefix=data_root + 'val2017/'))
```

Comment:
Is it just the public model from mmdetection?

Reply:
Yes.